ArcWelderPlugin.py

```diff
@@ -112,6 +113,14 @@ def _filterGcode(self, output_device):
         maximum_radius = global_container_stack.getProperty("arcwelder_maximum_radius", "value")
         tolerance = global_container_stack.getProperty("arcwelder_tolerance", "value") / 100
         resolution = global_container_stack.getProperty("arcwelder_resolution", "value")
+        min_arc = int(global_container_stack.getProperty("arcwelder_min_arc_segment", "value"))
+        mm_par_arc = global_container_stack.getProperty("arcwelder_mm_per_arc_segment", "value")
+
+        
+        cmd_3d_arcs = ""
+        allow_3d_arcs = global_container_stack.getProperty("arcwelder_allow_3d_arcs", "value")
+        if allow_3d_arcs:
+            cmd_3d_arcs = "-z"
```

Comment:
This could be written as 
`cmd_3d_arcs = "-z" if global_container_stack.getProperty("arcwelder_allow_3d_arcs", "value") else ""`